src/relay/limiter/fixed.rs

```diff
@@ -35,6 +38,9 @@ impl FixedWindow {
 
 #[async_trait]
 impl Metrics for FixedWindow {
+    /// feed returns `true` if a user hasn't exceeded his consumption limits, `false` otherwise.
+    /// if the current timestamp is different from the current counter start, the counter is reset to zero values, with the start equal to the current timestamp.
+    /// then, after incrementing the counters, the limits are checked, and a boolean value is returned accordingly.
     async fn feed(&self, size: usize) -> bool {
```

Comment:
I don't understand the reasoning behind the function name `feed`

Reply:
yeah, this might be my fault actually but since this is a metric aggregator i thought we will be `feeding` it different kind of data. can also be `track`, `measure` or something else